microsoftgraph/microsoft-graph-explorer-v4
# Fix: Update HAR file export from the History queries

## Overview

Fixes #1182 
Queries in the History action can now be exported and imported in Postman

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Open Graph Explorer
* Run a query
* Navigate to the History tab
* Select 'Export'
* Open Postman and import the exported .har file
* Observe fix
